_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,320 @@
+---
+id: be4d0c
+name: Inline link is distinguishable
+rule_type: composite
+description: |
+  This rule checks that inline links have a way to distinguish them from the surrounding text
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_rules:
+  - e7cc2b
+  - 88407d
+  - 36f116
+  - 66e9f0
+  - 548868
+  - 2803b8
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], where the closest element (including itself) that generates an [inline box][] is part of a [line box][] that has [inline boxes][inline box] generated from an element containing [visible][] [text nodes][text node] that do not have a [semantic link][] as a [descendant][] in the [flat tree][].
+
+**Note:** The test targets are visible link elements that are part of a block of content that is displayed inline.
+
+## Expectation
+
+For each test target, the [outcome][] of one of the following rules is "passed":
+
+- [Inline link has distinguishable content](https://act-rules.github.io/rules/e7cc2b)
+- [Inline link has distinguishable style not based on color alone](https://act-rules.github.io/rules/88407d)
+- [Inline link has distinguishable border](https://act-rules.github.io/rules/36f116)
+- [Inline link has distinguishable box-shadow](https://act-rules.github.io/rules/66e9f0)
+- [Inline link has different foreground color and distinguishable style on hover and focus](https://act-rules.github.io/rules/548868)
+- [Inline link has different background color and distinguishable style on hover and focus](https://act-rules.github.io/rules/2803b8)
+
+## Assumptions
+
+- This rule assumes that [`br` HTML elements][br] are only used for line breaks that are actually part of the content and not for separating thematic groups of the content. 
+- This rule assumes that the link is distinguishable from the rest of the text with color, which means it fails SC 1.4.1 when there is not another way to distinguish it.
```

Comment:
From what I recall, this is here to state that the link's text is distinguishable from the other text by color, since that is where SC 1.4.1 applies. @ShadowBB please correct me if I'm wrong.